docs/_index.yaml

```diff
@@ -1,23 +1,5 @@
-# Copyright 2020 The Cirq Developers. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# This file desribes the main splash page for the Cirq sub-site on the 
-# larger Google Quantum site. 
-# It is used by "devsite", a Google internal CMS platform for documentation. 
-
```

Comment:
Why does the license have to go? 
How about the "usage" comment - does that have to go too? 
Does devsite have issues with yaml comments?

Reply:
I'll fix and keep the license as-is. Your link OSS makes sense, thanks.
Re-adding usage section. That was accidentally removed when I copied over an earlier file.

Reply:
Ah great! Thank you!